src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/AllowedCharactersBitmap.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
hmm, I tried to git mv and rebase. It seemed to all work, but this still shows as deleted and added file. Maybe too much changed?